Source/JavaScriptCore/wasm/WasmBBQJIT.cpp

```diff
@@ -5733,13 +5733,13 @@ class BBQJIT {
         Jump tierUp = m_jit.branchAdd32(CCallHelpers::PositiveOrZero, TrustedImm32(TierUpCount::functionEntryIncrement()), Address(m_scratchGPR));
         MacroAssembler::Label tierUpResume = m_jit.label();
         auto functionIndex = m_functionIndex;
-        addLatePath([tierUp, tierUpResume, functionIndex](BBQJIT&, CCallHelpers& jit) {
+        bool isSIMD = m_isSIMD;
```

Comment:
We should get this when late path is executed. `notifyFunctionUsesSIMD` is not called yet when `emitEntryTierUpCheck` is called.

Reply:
It should be called before `addTopLevel`, so it'll be true if any of our arguments or locals are SIMD (which I think is all we need to handle entry tier-up correctly?). But I do agree it's a better, less fragile solution to move the capture (although capturing members is a pain). I'll fix it up.

Reply:
It is possible that body can include SIMD operations and it generates V128 temp (not local).

Reply:
Should be resolved in the latest push!